iocore/net/OCSPStapling.cc

```diff
@@ -20,11 +20,18 @@
  */
 
 #include "P_OCSPStapling.h"
+
 #if TS_USE_TLS_OCSP
 
 #include <openssl/bio.h>
 #include <openssl/ssl.h>
+
+#if TS_HAS_BORINGOCSP
+#include "boringocsp/ocsp.h"
```

Comment:
Are we assuming boringocsp/ocsp.h is in our tree?  Or why are we using double quotes instead of angle brackets here?  Not impacting functionality, but maybe confusing from the maintenance perspective.

Reply:
Good call! Fixed.